lib/lndclient/LndClient.ts

```diff
@@ -72,6 +73,25 @@ class LndClient extends BaseClient {
     });
   }
 
+  private connect = (lndCLient: LndClient, subscribeInvoices: boolean = false) => {
+    const { host, port } = lndCLient.config;
+
+    lndCLient.lightning = new LightningClient(`${host}:${port}`, lndCLient.credentials);
+
+    lndCLient.getInfo().then((getInfoResponse) => {
+      if (getInfoResponse && getInfoResponse.numActiveChannels >= 1) {
```

Comment:
Why do we need there to be at least 1 active channel to consider the connection verified? We can be connected to lnd even without any active channels.

Reply:
now just checking if there's a response from getInfo to determine the successful connection